cEP-0004.md

```diff
@@ -35,6 +35,24 @@ The following changes shall be made to the UI:
 
 The below sections describe why and how we should make these changes.
 
+### Mockup
+
+```
+test.py
+|   1| print(•1•)
+|    | [NORMAL] PEP8Bear:
+|    | The code does not comply to PEP8.
+|----|    | /tmp/test.py
+|    |++++| /tmp/test.py
+|   1|    |-print( 1 )
+|    |   1|+print(1)
+|    |
+|    | * Do (n)othing
+|    |   (O)pen file(s)
+|    |   (A)pply the patch
+|    | Choose an action:
```

Comment:
Also, after reading on issue 1021 about getting some parameters and generates a diff, I am thinking about a `profile` for a user. Let's say we implement that parameters( e.g variable renaming) and user A wants to use that variable everywhere when he run coala -> he make a `profile` let's say profileA.
It will be like a coafile(same syntax) but it will work in every directory(depending on permissions) with or without .coafile in it.